docs/products/images/deep_coadd.rst

```diff
@@ -38,7 +38,7 @@ A DP1 visit image must have a PSF FWHM < 1.7 arcseconds to be selected as an inp
 A mean stacking algorithm, weighted by inverse variance, combines selected exposures.
 
 Each individual deep coadd image covers a single patch of the sky:
-a quadrilateral sub-region of the overall skymap that covers approximately 79 square arcminutes.
+a quadrilateral sub-region of the overall skymap that covers approximately 79 square arcminutes, with a pixel scale of 0.2 arcseconds.
```

Comment:
- `pixel scale of 0.2 arcseconds` ---> `pixel scale of 0.2 arcseconds per pixel` 
-  Following on Melissa's comment in the Rubin Slack, I think having a bit more detail is fine. And I wonder now if we should add a statement about the pixel scale in the corresponding `visit_image` page, pointing out that it may vary with position and that, therefore, the `wsc.getPixelScale()` method should not be used with an empty argument, as Jim advised in a forum post and as we documented in the "Known issues" page of the DP1 documentation(which we can link to from that `visit_image` page if we do this addition).